routing/optional_placeholders.rst

```diff
@@ -11,7 +11,7 @@ the available blog posts for this imaginary blog application:
 
     .. code-block:: php-annotations
 
-        // src/AppBundle/Controller/BlogController.php
+        // src/App/Controller/BlogController.php
```

Comment:
The `App` in the file path was removed while merging, because it's not needed (it's only for the PHP namespace).